x-pack/legacy/plugins/integrations_manager/server/packages/handlers.ts

```diff
@@ -27,24 +27,19 @@ interface ListPackagesRequest extends Request {
   query: Request['query'] & SearchParams;
 }
 
-interface PackageRequest extends Request {
+interface PackageInfoRequest extends Request {
   params: {
     pkgkey: string;
   };
 }
 
-interface InstallAssetRequest extends Request {
-  params: AssetRequestParams;
-}
-
-interface DeleteAssetRequest extends Request {
-  params: AssetRequestParams;
+interface InstallDeletePackageRequest extends Request {
```

Comment:
I liked that these were separate and am 50/50 on the name but we're not exporting and we can always split them up later.

Reply:
Yeah, the important change here is `Asset` -> `Package`. I can split them, but I couldn't think (for now) of a situation where they would be different.